GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -692,6 +699,14 @@ private void writeAnswer(
         for (String cellValue : cells) {
             if (questionType == QuestionType.NUMBER) {
                 createCell(row, col, cellValue, null, Cell.CELL_TYPE_NUMERIC);
+            } else if (questionType == QuestionType.OPTION){
```

Comment:
this is more personal style issue but if you used..
 `(questionType == QuestionType.OPTION && (cellValue.equals("0") || cellValue.equals("1")))`
you would skip having the same line repeated on 708 and 711 below.